redaxo/src/addons/media_manager/pages/types.php

```diff
@@ -71,12 +71,54 @@
 }
 
 if ($func == '') {
+    echo '<style>
```

Comment:
CSS und JS ggf. auslagern in externe Datei.

Reply:
"Ggf." oder: "mach das!" ? ;-)

Reply:
würde ich persönlich besser finden :wink: